### PR TITLE
GEOPY-690

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-690 - Fix Z drape from topo optional**

### CODE REVIEW

File: geoapps/utils/models.py [hunk 1]
```diff
@@ -20,6 +25,26 @@
...
+
+
+def floating_active(mesh: TreeMesh, active: np.ndarray):
+    """
+    True if there are any active cells in the air
+
+    :param mesh: Tree mesh object
+    :param active: active cells array
+    """
+    return True if 6 in face_average(mesh, active) else False
```
Comment (domfournier) [inline geoapps/utils/models.py:45]: That's cool, didn't know we could do 6 in. No issues with integer and float array?

**Review by domfournier — APPROVED** [review]
Looking great. See comment below

File: geoapps/inversion/components/topography.py [hunk 2]
```diff
@@ -96,15 +97,30 @@ def active_cells(self, mesh: InversionMesh, data: InversionData) -> np.ndarray:
...
+
             active_cells[
                 mesh.mesh._get_containing_cell_indexes(  # pylint: disable=protected-access
                     data.locations
                 )
             ] = True
+
+            if floating_active(mesh.mesh, active_cells):
+                warnings.warn(
+                    "Active cell adjustment has created a patch of active cells in the air, likely due to a faulty survey location."
```
Comment (domfournier) [inline geoapps/inversion/components/topography.py:121]: Should probably unittest this part

File: tests/run_tests/driver_dc_test.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ def test_dc_fwr_run(
         geoh5=geoh5,
         mesh=model.parent.uid,
         topography_object=topography.uid,
-        z_from_topo=True,
+        z_from_topo=False,
         data_object=survey.uid,
         starting_model=model.uid,
         resolution=None,
```
Comment (domfournier) [inline tests/run_tests/driver_dc_test.py:59]: need to send `drape_height=0`, otherwise we are getting air cells

**Review by domfournier — CHANGES_REQUESTED** [review]

File: tests/run_tests/driver_dc_test.py (hunk 3 above)
Comment (benk-mira) [reply tests/run_tests/driver_dc_test.py:60]: done

File: geoapps/inversion/components/topography.py (hunk 2 above)
Comment (benk-mira) [reply geoapps/inversion/components/topography.py:121]: I wrote unit tests for the floating_active and face_average methods

**Pull request merged**